pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/bindings/LTensorTNCudaBindings.hpp

```diff
@@ -74,6 +74,18 @@ void registerBackendClassSpecificBindings(PyClass &pyclass) {
                 tensor_network.getData(data_ptr, state.size());
             },
             "Copy StateVector data into a Numpy array.")
+        .def(
+            "updateMPSSites",
+            [](TensorNet &tensor_network, std::vector<np_arr_c> &sites) {
+                for (std::size_t idx = 0; idx < sites.size(); idx++) {
+                    py::buffer_info numpyArrayInfo = sites[idx].request();
+                    auto *data_ptr = static_cast<std::complex<PrecisionT> *>(
+                        numpyArrayInfo.ptr);
+                    tensor_network.updateIthMPSSite(idx, data_ptr,
+                                                    sites[idx].size());
```

Comment:
We updating the sites data and I would expect `sites` to be indices or tags. Could we change to, say, `tensors` since this is a tensor network? 
```suggestion
            [](TensorNet &tensor_network, std::vector<np_arr_c> &tensors) {
                for (std::size_t idx = 0; idx < tensors.size(); idx++) {
                    py::buffer_info numpyArrayInfo = tensors[idx].request();
                    auto *data_ptr = static_cast<std::complex<PrecisionT> *>(
                        numpyArrayInfo.ptr);
                    tensor_network.updateIthMPSSite(idx, data_ptr,
                                                    tensors[idx].size());
```